src/test/java/core/basesyntax/service/impl/FruitShopServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class FruitShopServiceImplTest {
+    private final FruitShopService<FruitTransaction> fruitShopService = new FruitShopServiceImpl(
+            new OperationStrategyImpl());
+
+    @Test
+    void update_validList_Ok() {
+        Storage.getStorage().clear();
```

Comment:
Move to beforeEach do it everywhere. This is necessary if some new logic is added and, accordingly, there will be more tests, and in order for everything to be beautiful for each test, you need to clear the database